setup.py

```diff
@@ -0,0 +1,31 @@
+import setuptools
+
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="tiledb_netcdf",
+    version="0.1.0",
+    author="Peter Killick",
+    author_email="peter.killick@informaticslab.co.uk",
+    description="An adapter to convert NetCDF files to TileDB or Zarr arrays",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/informatics-lab/tiledb_netcdf",
+    packages=setuptools.find_packages(),
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+        "Operating System :: OS Independent",
+    ],
+    python_requires=">=3.7",
+    install_requires=[
+        "netCDF4>=1.5.3",
+        "tiledb>=0.6.2",
```

Comment:
The latest updates I've made mean we could probably bump this dependency now: 
```suggestion
        "tiledb>=0.6.6",
```